omnikinverter/omnikinverter.py

```diff
@@ -76,9 +76,13 @@ async def request(
             "Accept": "text/html,application/xhtml+xml,application/xml",
         }
 
-        if self._session is None:
-            self._session = ClientSession()
-            self._close_session = True
+        # if self._session is None:
+        #     self._session = ClientSession()
+        #     self._close_session = True
+
+        # Don't reuse session as Wifi stick does not seem to support it (on some firmware versions?)
+        self._session = ClientSession()
+        self._close_session = True
```

Comment:
Ah, yes. That makes sense. I now also see the warnings in my log (although HA does not clearly indicate the source)
Since it is not using the session anymore I will just go ahead and cleanup all the session related stuff. No reason to keep it around.